roles/st2chatops/meta/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+galaxy_info:
+  description: Install st2chatops
+  author: humblearner
+  company: StackStorm
+  license: Apache
+  min_ansible_version: 2.2
+  platforms:
+    - name: Ubuntu
+      versions:
+        - trusty
+        - xenial
+    - name: EL
+      versions:
+        - 6
+        - 7
+  categories:
+    - st2
+    - devops
+    - chatops
+    - autmation
+dependencies:
+    - role: st2
```

Comment:
`st2` is not a requirement for `st2chatops` since `st2chatops` can be installed on a separated machine, it's possible to configure [remote API endpoints](https://github.com/StackStorm/st2chatops/blob/master/st2chatops.env#L22-L26) and it will work.

Saying that, `st2repo` is a dependency for sure in install st2chatops package.

---

cc @lakshmi-kannan you asked before why we need `st2repo` and `st2` roles separated.